indexbuilder.go

```diff
@@ -187,10 +189,10 @@ type IndexBuilder struct {
 	subRepoIndices []map[string]uint32
 
 	// language => language code
-	languageMap map[string]byte
+	languageMap map[string]uint16
 
-	// languages codes
-	languages []byte
+	// language codes, uint16 encoded as little-endian
+	languages []uint8
```

Comment:
I don't really understand why this isn't just uint16? You don't need backwards compatibility here since you just write this out and do not read.

Reply:
do you mean for when reading the older disk formats? Or you just wanna be explicit in the little endian encoding? This array is purely for the encoding, somewhere else we actually do the marshalling. The indexData.languages is still a []byte which I understand since that is how we get data back from our mmap thing.